x/application/client/cli/helpers_test.go

```diff
@@ -24,6 +26,8 @@ func networkWithApplicationObjects(t *testing.T, n int) (*network.Network, []typ
 	t.Helper()
 	cfg := network.DefaultConfig()
 	appGenesisState := network.DefaultApplicationModuleGenesisState(t, n)
-	network.HydrateGenesisState(t, &cfg, appGenesisState, types.ModuleName)
+	buf, err := cfg.Codec.MarshalJSON(appGenesisState)
```

Comment:
If we're not going to use `HydrateGenesisState`, let's remove that funciton.

Reply:
Pretty sure I did